pkg/collector/python/init.go

```diff
@@ -290,6 +293,50 @@ func pathToBinary(name string, ignoreErrors bool) (string, error) {
 	return absPath, nil
 }
 
+func getPythonPackagesVersion(pythonBinPath string) map[string]string {
+	args := []string{
+		"-m",
+		"pip",
+		"freeze",
+	}
+	pipCmd := exec.Command(pythonBinPath, args...)
```

Comment:
The result of `pip freeze` which can be massive. We have limitation regarding payload size imposed by the intake. We're also need to be mindful of the size in REDAPL.

Could we filter the result to only include python package not ship by default with the Agent ?

Reply:
Related to https://github.com/DataDog/datadog-agent/pull/34320#discussion_r1969764368
This helps us having a reliable source of what's currently running within the agent, without merging multiple sources and try to correct error of what kind go wrong (what happens for marketplace integration? If a user upgraded a datadog dep manually?)